libs/user_extensions/src/TTAlpsObjetcsManager.cpp

```diff
@@ -0,0 +1,84 @@
+#include "TTAlpsObjectsManager.hpp"
```

Comment:
There's a typo in the file name "TTAlpsObjetcsManager.cpp" -> "TTAlpsObjectsManager.cpp"